WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -458,6 +474,22 @@ private void getMediaFromDeviceAndTrack(Uri imageUri, int requestCode) {
         );
     }
 
+    private void checkRecordedVideoSizeBeforeUploadAndTrack() {
```

Comment:
Are we checking size, or did we change to duration? Wondering if we should rename this `s/Size/Duration` in the method name. Wdyt?